### PR TITLE
Updated minRatio verbiage

### PR DESCRIPTION
**Pull request by hecktarzuli**
This PR contains a:

- [ ] **bugfix**
- [ ] new **feature**
- [ ] **code refactor**
- [ ] **test update** 
- [x] **typo fix**
- [ ] **metadata update**

### Motivation / Use-Case

https://github.com/webpack-contrib/compression-webpack-plugin/issues/130

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -220,7 +220,7 @@ Default: `0.8`
 Only assets that compress better than this ratio are processed (`minRatio = Compressed Size / Original Size`).
 Example: you have `image.png` file with 1024b size, compressed version of file has 768b size, so `minRatio` equal `0.75`.
 In other words assets will be processed when the `Compressed Size / Original Size` value less `minRatio` value.
-You can use `1` value to process all assets.
+You can use `1` value to process assets that are smaller than the original.  Use a value of `999` to process all assets even if they are larger than the original (useful when you are pre-zipping all assets for AWS)
```
Comment (evilebottnawi) [inline README.md]: Maybe better `MATH.MAX_SAFE_INTEGER`?

Comment (hecktarzuli) [reply README.md]: Lol, sure, why not :) . (Number.MAX_SAFE_INTEGER)

**Comment by evilebottnawi** [comment]
Can you update lock file to avoid problems with audit?

**Comment by hecktarzuli** [comment]
Sorry, I don't follow.  I just updated the README.md, what lock file and why?

**Comment by evilebottnawi** [comment]
@hecktarzuli CI failed because security problem in node_modules, need regenerate https://github.com/webpack-contrib/compression-webpack-plugin/blob/master/package-lock.json, just rm -r package-lock.json and npm i again and send a new lock file in this PR

**Comment by hecktarzuli** [comment]
Looks like it failed again.  Honestly, I've already spent way too much time on this.  I just wanted to help you out and update your doc but this has being a pain in the butt.  Do what you want with this pull.

**Pull request merged**